app/controllers/gallery/albums_controller.rb

```diff
@@ -0,0 +1,12 @@
+# encoding:UTF-8
+class Gallery::AlbumsController < ApplicationController
+  load_permissions_and_authorize_resource
+
+  def show
+    if params[:start].present?
+      @start = params[:start]
+    elsif params[:image].present?
+      @start = @album.images.find_index(Image.find(params[:image]))
+    end
```

Comment:
It's to be used as:
`galleri/album/1/start=10`
Will start the javascript at image 10 in loaded order.
`galleri/album/1/image=10`
Will start the javascript at image with id 10.